CIPs/CIP-85/CIP-85.md

```diff
@@ -111,18 +111,20 @@ Accessing slices in the collection relies on Ceramic's ability to load documents
 
 > This flow assumes a prerequisite check that the `AppendCollection` document has not been created yet.
 
-1. Create the `CollectionSlice` document with the `contents` array containing the item to insert.
+1. Create a deterministic `CollectionSlice` document with `sliceNumber` of `0` and an empty `contents` array.
+1. Update the created `CollectionSlice` document with the `contents` array containing the item to insert.
 1. Create the `AppendCollection` document with a `slicesCount` of `1`.
```

Comment:
hmm... I just realized it may be difficult to create the `CollectionSlice` before the `AppendCollection` since you need the DocID from the `AppendCollection` in order to make the `CollectionSlice`

Reply:
👍 right, good catch! I updated to start the flow with the `AppendCollection` document creation